screens/EmailVerification.js

```diff
@@ -40,11 +40,6 @@ function EmailVerification({ route, navigation }) {
   }
 
   async function resendCode() {
-    if (email == "") {
```

Comment:
We need this block here, otherwise Expo loads another error message since the Auth call throws an error that we somehow don't catch.